frontend/src/Admin/components/ShowSubmissionsTab.js

```diff
@@ -15,17 +15,24 @@ import { getImageThumbnail } from '../../utils'
 const SUCCESS = 'SUCCESS'
 const ERROR = 'ERROR'
 
+// Invite button Message
+const FINALIZE_INVITES = 'Make invites public'
```

Comment:
I don't think you really need to make constants for these.

And I think they should word:
"Make Invitations Public"
"Invitations Are Public"